tests/mochainon.d.ts

```diff
@@ -0,0 +1,12 @@
+declare module 'mochainon' {
+	import * as chai from 'chai';
+	import * as sinon from 'sinon';
+
+	interface Mochainon {
+		chai: typeof chai;
+		sinon: typeof sinon;
+	}
+
+	const mochainon: Mochainon;
+	export = mochainon;
+}
```

Comment:
Bonus argument for `resin-io/types-*`: it'd be much easier to deduplicate things like this from across all our projects